### PR TITLE
refactor: move isAuthenticated from context to zustand store"

### PR DESCRIPTION
**Pull request by anovazzi1**
This pull request moves the `isAuthenticated` state from the `AuthContext` to the `authStore` in order to centralize the authentication logic. This change improves code organization and makes it easier to manage the authentication state throughout the application.

### CODE REVIEW

File: src/frontend/src/stores/authStore.ts [hunk 1]
```diff
@@ -19,8 +20,8 @@ const cookies = new Cookies();
 
 const useAuthStore = create<AuthStoreType>((set, get) => ({
   isAdmin: false,
-  isAuthenticated: !!cookies.get("access_token_lf"),
-  accessToken: cookies.get("access_token_lf") ?? null,
+  isAuthenticated: !!!!cookies.get(LANGFLOW_ACCESS_TOKEN),
```
Comment (Cristhianzl) [inline src/frontend/src/stores/authStore.ts:23]: just a question, this:
isAuthenticated: !!cookies.get(LANGFLOW_ACCESS_TOKEN) 
is not supposed to have the same behavior?

**Review by Cristhianzl — CHANGES_REQUESTED** [review]

File: src/frontend/src/stores/authStore.ts (hunk 1 above)
Comment (anovazzi1) [reply src/frontend/src/stores/authStore.ts:23]: yes, i don't know why i did this lol

Comment (anovazzi1) [reply src/frontend/src/stores/authStore.ts:23]: i will fix it now

**Review by Cristhianzl — APPROVED** [review]
LGTM

**Pull request merged**